src/main/java/seedu/address/model/ReadOnlyAddressBook.java

```diff
@@ -28,4 +29,10 @@
     ObservableList<Tag> getTagList();
 
     DebtsTable getDebtsTable();
+
+    ObservableList<Debtor> getDebtorsList();
+
+    void setDebtors(DebtsList debtsList);
+
+    void setCreditors(DebtsList debtsList);
```

Comment:
I don't think set methods should be in the "ReadOnlyAddressbook".

Reply:
the getAddress() method is inside modelmanager is get readonlyaddressbook type.